examples/advanced_operations/add_ad_group_asset.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example adds an ad group asset.
+
+To upload image assets for this campaign, run misc/upload_image_asset.py.
```

Comment:
Remove reference to campaign. You upload image assets and then link them to an entity. You don't upload assets for a campaign. The upload_image_asset.py example is just for uploading assets and not linking them.

Reply:
Changed.